core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -295,27 +350,40 @@ public MultiMatchQueryBuilder cutoffFrequency(float cutoff) {
         return this;
     }
 
+    /**
+     * Set a cutoff value in [0..1] (or absolute number >=1) representing the
+     * maximum threshold of a terms document frequency to be considered a low
+     * frequency term.
+     */
+    public MultiMatchQueryBuilder cutoffFrequency(Float cutoff) {
+        this.cutoffFrequency = cutoff;
+        return this;
+    }
+
+    public MultiMatchQueryBuilder zeroTermsQuery(ZeroTermsQuery zeroTermsQuery) {
+        this.zeroTermsQuery = zeroTermsQuery;
+        return this;
+    }
 
+    @Deprecated
     public MultiMatchQueryBuilder zeroTermsQuery(MatchQueryBuilder.ZeroTermsQuery zeroTermsQuery) {
-        this.zeroTermsQuery = zeroTermsQuery;
+        this.zeroTermsQuery = ZeroTermsQuery.valueOf(zeroTermsQuery.name().toUpperCase(Locale.ROOT));
```

Comment:
fyi, I removed the ZeroTermsQuery class from MatchQueryBuilder entirely in #13402, moved everything plus serialization to #13402, so we might remove this setter and only have one.